test/system/avo/rubygems_test.rb

```diff
@@ -26,9 +26,11 @@ def sign_in_as(user)
     admin_user = create(:admin_github_user, :is_admin)
     sign_in_as admin_user
 
-    rubygem = create(:rubygem)
+    rubygem = create(:rubygem, created_at: 40.days.ago)
```

Comment:
This actually places the gem into the state where it's protected, rather than having the test release the namespace when it wasn't protected to begin with.